source/agora/common/Config.d

```diff
@@ -138,6 +138,9 @@ public struct NodeConfig
 
     /// Maximum number of nodes to include in an autogenerated quorum set
     public uint max_quorum_nodes = 7;
+
+    /// Threshold to use in the autogenerated quorum
+    public double quorum_threshold = 0.80;
```

Comment:
It's only for the unittests. In this case it makes it significantly easier to write a simple test-case - otherwise I would have to instantiate many more nodes and I prefer to have tests which are simple to understand.

I don't actually want this to be in the node configs. But there was a commit which removed the `ConsensusParams` from the `FullNode` ctor parameter list and `ConsensusParams` was instead instantiated directly in the constructor body using the node config parameters.